.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ocaml.sandbox": {
+    "root": "/Users/Jaap/Documents/development/graphql_ppx_re",
```

Comment:
This is user specific. I don't think it should be committed.

Reply:
whoops 🙃